migration-guide/src/explorer-api.md

```diff
@@ -420,8 +420,14 @@ Get block's summary information.
 
 === "cardano-rosetta"
 
-    !!! warning
-        Not available in Rosetta!
+                    curl -X POST http://localhost:8080/block \
+                    -H "Content-Type: application/json" \
+                    -d '{ "network_identifier": {
+                    "blockchain": "cardano", 
+                    "network": "mainnet" },
+                    "metadata": {},
+                    "block_identifier": {
+                    "index": "$INDEX" }}' | jq
```

Comment:
Do we also have examples of responses from cardano-rosetta here? So that it's easier to compare results; what's present and what's not.

Reply:
I have updates the cardano-rosetta API with sample responses